xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v1/service/BitfinexMarketDataServiceRaw.java

```diff
@@ -71,14 +73,22 @@ public BitfinexLendDepth getBitfinexLendBook(String currency, int limitBids, int
   }
 
   public List<CurrencyPair> getExchangeSymbols() throws IOException {
-    List<CurrencyPair> currencyPairs = new ArrayList<>();
-    for (String symbol : bitfinex.getSymbols()) {
-      currencyPairs.add(BitfinexAdapters.adaptCurrencyPair(symbol));
+    try {
+      List<CurrencyPair> currencyPairs = new ArrayList<>();
+      for (String symbol : bitfinex.getSymbols()) {
+        currencyPairs.add(BitfinexAdapters.adaptCurrencyPair(symbol));
+      }
+      return currencyPairs;
+    } catch (BitfinexException e) {
```

Comment:
this try / catch should be done only in generic methods
not in *Raw methods

https://github.com/knowm/XChange/wiki/New-Implementation-Best-Practices#step-7